lib/config/presets/internal/monorepo.ts

```diff
@@ -544,6 +544,7 @@ const orgGroups = {
   ngrx: 'https://github.com/ngrx/',
   nrwl: 'https://github.com/nrwl/',
   octokit: 'https://github.com/octokit/',
+  pino: 'https://github.com/pinojs/',
```

Comment:
not a good idea, they have different versions, so those PRs are immortal.

@rarkins we should revert this

Reply:
Octokit has different versions for it's packages, so by that logic it should also be reverted.